AudioAnalytics/AudioAnalyzer/src/main/java/sample/AudioAnalyzer.java

```diff
@@ -114,50 +108,37 @@ private static void runAnalyzer(ConfigWrapper config) {
                     outputAssetName);
 
             Job job = submitJob(manager, config.getResourceGroup(), config.getAccountName(),
-                audioAnalyzerTransform.name(), jobName, inputAssetName, outputAsset.name());
+                    audioAnalyzerTransform.name(), jobName, inputAssetName, outputAsset.name());
 
             long startedTime = System.currentTimeMillis();
-            
-            EventProcessorHost eventProcessorHost = null;
+
             try {
                 // First we will try to process Job events through Event Hub in real-time. If this fails for any reason,
                 // we will fall-back on polling Job status instead.
                 System.out.println("Creating an event processor host to process events from Event Hub...");
+                String storageConnectionString = config.getStorageConnectionString();
 
-                String storageConnectionString = "DefaultEndpointsProtocol=https;AccountName=" +
-                    config.getStorageAccountName() +
-                    ";AccountKey=" + config.getStorageAccountKey();
-                
                 // Cleanup storage container. We will config Event Hub to use the storage container configured in appsettings.json.
                 // All the blobs in <The container configured in appsettings.json>/$Default will be deleted.
-                CloudStorageAccount account = CloudStorageAccount.parse(storageConnectionString);
-                CloudBlobClient client = account.createCloudBlobClient();
-                CloudBlobContainer container = client.getContainerReference(config.getStorageContainerName());
-                for (ListBlobItem item : container.listBlobs("$Default/", true)) {
-                    if (item instanceof CloudBlob) {
-                        CloudBlob blob = (CloudBlob) item;
-                        blob.delete();
-                    }
-                }
+                BlobServiceAsyncClient client = new BlobServiceClientBuilder()
+                        .connectionString(storageConnectionString)
+                        .buildAsyncClient();
+                BlobContainerAsyncClient container = client.getBlobContainerAsyncClient(config.getStorageContainerName());
+                container.listBlobs(
+                        new ListBlobsOptions().setPrefix("$Default/"))
```

Comment:
Have you tried this code? We don't create blobs starting with the `$Default/` prefix in Track 2.

Reply:
According to your comments, it has been fixed in the new version.